packages/core/status/core-status-server-internal/src/routes/status.ts

```diff
@@ -100,21 +88,43 @@ export const registerStatusRoute = ({
         // ROUTE_TAG_ACCEPT_JWT from '@kbn/security-plugin/server' that cannot be imported here directly.
         tags: ['api', 'security:acceptJWT'],
         access: 'public', // needs to be public to allow access from "system" users like k8s readiness probes.
+        description: `Get Kibana's current status.`,
```

Comment:
>        description: `Get Kibana's current status.`,

In the current output, I see this string used in the operation summary. This is indeed the more critical field from a docs point of view, however we'll ultimately want to have both a summary and an (optional) description. Per https://spec.openapis.org/oas/latest#operation-object, `summary`  is "A short summary of what the operation does" and `description` is "A verbose explanation of the operation behavior. CommonMark syntax MAY be used for rich text representation". For example, you can see the different details I think we'd put in those areas in https://github.com/elastic/kibana/blob/0c0cf0a8e0e9dde31f9c9872cde328984065dfc8/x-pack/plugins/actions/docs/openapi/bundled.json#L279

summary	string	A short summary of what the operation does.
description	string	A verbose explanation of the operation behavior. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
summary | string | A short summary of what the operation does.
-- | -- | --
description | string | A verbose explanation of the operation behavior. CommonMark syntax MAY be used for rich text representation.

summary	string	A short summary of what the operation does.
description	string	A verbose explanation of the operation behavior. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.